Lib/test/test_future.py

```diff
@@ -1,5 +1,7 @@
 # Test various flavors of legal and illegal future statements
 
+import __future__
```

Comment:
Is there a risk that this import change the behavior to test_future? I don't think so, but I ask, just in case.

Reply:
Don't think so. It is only effective in from imports.